csrc/flash_attn/src/flash_fwd_split_hdim224_fp16_sm80.cu

```diff
@@ -1,7 +0,0 @@
-// Copyright (c) 2023, Tri Dao.
-// Splitting the different head dimensions to different files to speed up compilation.
```

Comment:
There wasn't a config for 224 in the gen configs, so deleted but need to double check if needed

Reply:
> There wasn't a config for 224 in the gen configs, so deleted but need to double check if needed